static/fonts/style.css

```diff
@@ -0,0 +1,120 @@
+/* #### Generated By: http://www.cufonfonts.com #### */
```

Comment:
Hey @lennessyy, not sure if I understand why there are so many different BAHNSCHRIFT.woff files? They seems to be the same.

Reply:
I downloaded them online and assumed they they were different weights or something without looking into them deeper. Just deleted the duplicates.